pyspark-iris/{{ cookiecutter.repo_name }}/src/requirements.txt

```diff
@@ -4,10 +4,11 @@ ipython>=7.31.1, <8.0
 isort~=5.0
 jupyter~=1.0
 jupyterlab~=3.0
-kedro[spark.SparkDataSet]=={{ cookiecutter.kedro_version }}
+kedro[spark.SparkDataSet, pandas.CSVDataSet]=={{ cookiecutter.kedro_version }}
 kedro-telemetry~=0.2.0
 nbstripout~=0.4
 numpy~=1.21
 pytest-cov~=3.0
 pytest-mock>=1.7.1, <2.0
 pytest~=6.2
+PyArrow>=1.0.0
```

Comment:
Is this definitely needed? The kedro dataset dependencies should already install the right version of pyarrow.

Reply:
You are correct the dependencies install this.